tox.ini

```diff
@@ -3,6 +3,14 @@ envlist = qa,py36,py37,py38,py39,py310
 skip_missing_interpreters = True
 skipsdist = True
 
+[gh-actions]
```

Comment:
Please no tox-gh-actions crap. This is huge maintenance burden for no gain.

Reply:
Ok, what should I do instead?

Reply:
`tox -e py`

Reply:
You want a `tox` invocation per env, rather than a list of envs maintained in `tox.ini`? I'm not sure that's a lower maintenance burden.

Reply:
No clue what you're talking about. `py` runs the testenv using current Python. Definitely better than dumbly mapping every single Python version ever created to identical tox testenvs.

Reply:
It seems like Actions doesn't support 3.6 anymore, so that target couldn't be added, but Actions are now working from the perspective of my fork: https://github.com/ajakk/eclean-kernel/actions/runs/3587301428